### ISSUE TITLE
Error when -Syu : maximum file size exceeded

### ISSUE THREAD

**Issue by Lafolie** [jstkdng/aur#14]
Hi, I'm getting the following errors when trying to upgrade with this repo:
```
error: failed retrieving file 'ungoogled-chromium-92.0.4515.131-1-x86_64.pkg.tar.zst' from ftp.gwdg.de : Maximum file size exceeded
error: failed retrieving file 'ungoogled-chromium-92.0.4515.131-1-x86_64.pkg.tar.zst' from download.opensuse.org : Maximum file size exceeded
warning: failed to retrieve some files
error: failed to commit transaction (download library error)
Errors occurred, no packages were upgraded.
```
I've re-init packey, purged cache, and even tried adding a mirror, nothing seems to work. I read in a thread for a similar issue that it is due to incorrect details in the repo database, so I thought I'd open a ticket.

**Comment by Lafolie** [comment]
Actually, I just checked the opensuse page, might it be because it's building right now? Apologies if so, never encountered this error before.

**Comment by jstkdng** [comment]
is this still a problem?

**Comment by Lafolie** [comment]
Hi, sorry for the delay.
Yes, I still get the exact same error message reported. All of my other packages can update, it's just the ungoogled-chromium one from your repo that's failing.
I've also tried with just `pacman -S ungoogled-chromium` which gives the same error.

Edit:
Managed to sync the package by using the following command:
`pacman -Sy home_justkidding_arch_Arch/ungoogled-chromium`
I'm wondering whether it was due to the recent host switch? I had previously sync'd form the old host, perhaps manually specifying the source repo was the key. I actually found that line afterwards at this page: https://software.opensuse.org//download.html?project=home%3Ajustkidding%3Aarch&package=ungoogled-chromium which is indentical to your listed instructions except for that one line.
